samples/oauth_rails_vethealth/app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
```

Comment:
Is this used anywhere, and if so should this be changed? Or is this just default boilerplate stuff?

Reply:
this is boilerplate stuff.  I'm not using mailers anywhere